frontend/src/components/AccountReceiveTableWithdrawConfirmation.vue

```diff
@@ -108,7 +108,7 @@ import { UserAnnouncement } from '@umbra/umbra-js';
 import { FeeEstimate } from 'components/models';
 import { formatAddress } from 'src/utils/address';
 import { BigNumber, formatUnits } from 'src/utils/ethers';
-import { getEtherscanUrl, getGasPrice, round } from 'src/utils/utils';
+import { getEtherscanUrl, getGasPrice, round, roundTokenAmount } from 'src/utils/utils';
```

Comment:
_(arbitrary comment location to start a thread)_

Just documenting that we said the scope of this PR is not limited to Polygon, but in practice Polygon is where we'll be added support for more tokens first